### ISSUE TITLE
About the training iterations on cocostuff-10k

### ISSUE THREAD

**Issue by hsfzxjy** [shipra25jain/ESSNet#4]
I am trying to train ESSNet on cocostuff-10k (not COCO+LVIS), but could hardly find the total training iterations in the paper. 

I've tried training deeplabv3plus-resnet50 for 40k iterations, with other settings same as in the paper (bs=10, d=12, k=7), but only got 27.6 mIoU (which should be ~32 mIoU). Should I use longer training schedule?

Thanks!

**Comment by shipra25jain** [comment]
Hi @hsfzxjy ,
yes, our model requires a slightly more number of epochs to converge. I do not remember an exact number of iterations we trained the cocostuff10k dataset. And I do not have the access to that machine now. I am sure you can reproduce results from the paper if you train a bit longer. I would say try at least 100k iterations and make sure learning rate in the scheduler is not very low.

**Comment by hsfzxjy** [comment]
@shipra25jain Thanks. I will try 100k.
